src/main/resources/application.properties

```diff
@@ -7,7 +7,7 @@ spring.datasource.driver-class-name=org.postgresql.Driver
 # TODO (racansky, 2023-03-02, https://github.com/orgs/wultra/projects/29/views/1?pane=issue&itemId=21830474) PoC, revert it if not working or propagate to all components
 spring.datasource.hikari.auto-commit=false
 
-spring.jpa.database-platform=org.hibernate.dialect.PostgreSQL10Dialect
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
```

Comment:
This dialect is deprecated, I think it is better to keep the original dialect.

Reply:
Yes, it is deprecated at Hibernate 5. But in Hibernate 6 (Spring Boot 3), the deprecated class is `PostgreSQL10Dialect`.